### PR TITLE
fees: min multiplier definition and testing

### LINKED ISSUE #154: Define LenghtFeeAdjustments Parameters
LenghtFeeAdjustment requires 4 parameters to update `targeted_length_fee_adjustment` at the end of each block. The value is used in the following formula:

```inclusion_fee = base_fee + [targeted_length_fee_adjustment * length_fee] + [targeted_weight_fee_adjustment * weight_fee];```

It requires 4 parameters:
- Target
- AdjustmentVariable
- MinimumMultiplier
- MaximumMultiplier

The current values for `targeted_weight_fee_adjustment` are:
- TargetBlockFullness = 25%
- AdjustmentVariableBlockFullness = 75/1_000_000
- MinimumMultiplierBlockFullness = 1/10
- MaximumMultiplierBlockFullness = FixedU128::max_value()

These values are defined [here](https://research.web3.foundation/Polkadot/overview/token-economics#2-slow-adjusting-mechanism) to define `SlowAdjustingFeeUpdate`.

For `targeted_length_fee_adjustment`, the values are:
- TargetBlockSize = 16%
   - If the total block size is 5MiB, then the target block size is 0.8MiB.
- AdjustmentVariableBlockSize = 1/840
   - This value means that the maximum fee variation in one hour is 30%, following the formula `v = p / k * (1 - s*)`, where p represents the maximum fee variation percentage, k represents the period in which the variation is defined (in blocks), and s* represents the target percentage.
- MinimumMultiplierBlockSize = TODO
- MaximumMultiplierBlockSize = FixedU128::max_value()

### PR DESCRIPTION
**Pull request by gabriele-0201**
closes #154

### CODE REVIEW

**Comment by gabriele-0201** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/182?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #174** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/174?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #175** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/175?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #176** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/176?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #181** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/181?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #182** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/182?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/thrumdev/blobs/182?utm_source=stack-comment).

**Review by rphmeier — APPROVED** [review]

File: sugondat/chain/runtimes/test/tests/fee_adjustments.rs [hunk 1]
```diff
@@ -244,6 +236,10 @@ fn multiplier_cannot_go_below_limit() {
 
 #[test]
 fn time_to_reach_zero() {
+    // TODO: 1440 needs to ne used insted of 7200
```
Comment (rphmeier) [inline sugondat/chain/runtimes/test/tests/fee_adjustments.rs:239]: probably meant: "be used instead of"

Comment (gabriele-0201) [reply sugondat/chain/runtimes/test/tests/fee_adjustments.rs:239]: Yes, for sure! Thanks!

**Comment by rphmeier** [comment]
### Merge activity

* **Jan 22, 1:25 PM**: @rphmeier started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/thrumdev/blobs/182).

**Pull request merged**